components/CaseStudiesList.vue

```diff
@@ -14,6 +14,20 @@
       </div>
     </div>
 
+    <div class="flex flex-wrap pt-12 pb-12 w-full lg:pr-4 lg:w-1/2">
+      <div class="m-auto max-w-2xl w-full lg:m-0">
+        <AcquiaLogo class="h-20 mb-3 w-auto" />
```

Comment:
```suggestion
        <AcquiaLogo class="h-10 mb-3 w-auto" />
```

I think all logos are `h-10`

Reply:
this makes it tiny, did you run the branch to see?

Reply:
It has to be `h-10` to all line up.